ZAPD/GameConfig.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <map>
+#include "Utils/Directory.h"
+#include "tinyxml2.h"
+
+struct TexturePoolEntry
+{
+	fs::path path = "";  // Path to Shared Texture
+};
+
+class ZFile;
+
+class GameConfig
+{
+public:
+	std::string configFilePath;
+	std::map<int32_t, std::string> segmentRefs;
```

Comment:
`<string>` and `<vector>` should probably be included too. And `<cstdint>` too for int32_t and uint32_t.